stripe/api_resources/account_link.py

```diff
@@ -5,7 +5,7 @@
 from stripe.api_resources.abstract import CreateableAPIResource
 
 
-class AccountLink(CreateableAPIResource):
+class AccountLink(CreateableAPIResource["AccountLink"]):
```

Comment:
Wrong quotes?

Reply:
Oh 🤦 looks like our formatter converts these. I'll change codegen to match what our formatter is doing.